openvalidators/prompts.py

```diff
@@ -348,16 +348,16 @@ def find_unique_tags(input_text: str):
 
 def followup_prompt( base_text:str, i:int = 0) -> str:
     if i == 0:
-        return f"{base_text}\n\n{followup_request_template}\n"
+        return f"{base_text}\n\n{followup_request_template}. Do not return an answer:\n"
     else:
-        return f"{base_text}\n\n{followup_request_template} and previous questions\n"
+        return f"{base_text}\n\n{followup_request_template} and previous questions. Do not return an answer:\n"
```

Comment:
Don't think you need to put if / else structure. Just this would be cleaner:

```
if i == 0:
         return f"{base_text}\n\n{followup_request_template}. Do not return an answer:\n"
return f"{base_text}\n\n{followup_request_template} and previous questions. Do not return an answer:\n"
```

Reply:
The if/else clause was alreadybthere, did not alter that.